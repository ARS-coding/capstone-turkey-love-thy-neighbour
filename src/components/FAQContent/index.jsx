import React from "react";

import { useTranslation } from "react-i18next";

import { Container, Row, Col } from "react-bootstrap";

import questionMarkImg from "../../images/UndrawQuestion.png";

import "./index.scss";

function FAQContent() {
  const { t, i18n } = useTranslation();
  return (
    <Container className="faq-content-container d-flex align-items-center">
      <Row className="faq-content-row m-0">
        <Col
          xs={12}
          sm={12}
          md={6}
          className="first-column d-flex flex-column justify-content-center align-items-start"
        >
          <h2>{t("global_FAQ")}</h2>
          <h5>{t("global_safety")}</h5>
          <p>{t("global_safety-text")}</p>
          <h5>{t("global_the_point")}</h5>
          <p>{t("global_the_point_text")}</p>
          <h5>{t("global_how_it_works")}</h5>
          <p>{t("global_how_it_works_text")}</p>
        </Col>

        <Col
          xs={12}
          sm={12}
          md={6}
          className="my-md-auto d-flex d-sm-flex d-md-block justify-content-center justify-content-sm-center justify-content-md-center"
        >
          <img src={questionMarkImg} alt="img" className="img-fluid" />
        </Col>
      </Row>
    </Container>
  );
}

export default FAQContent;
