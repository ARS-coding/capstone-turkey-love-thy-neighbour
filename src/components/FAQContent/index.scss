@import "../../style/variables";

.faq-content-container {
  height: 100%;

  .faq-content-row {
    & > div {
      padding: 0 !important;

      h2 {
        color: $logo-red;
        font-weight: bold;
      }

      h5 {
        color: $logo-blue;
      }

      p {
        color: $black;
        @media screen and (min-width: 993px) {
          font-size: 15px;
        }
        @media screen and (max-width: 252px) {
          font-size: 10px;
        }
      }

      img {
        @media screen and (min-width: 1200px) {
          width: 100%;
          height: 450px !important;
        }
        @media screen and (max-width: 495px) {
          width: 60%;
          height: 130px;
        }
        @media screen and (max-width: 340px) {
          display: none;
        }
      }
    }
  }
}

@media screen and (min-width: 768px) and (max-width: 992px) {
  .faq-content-row p {
    font-size: 13px;
  }
}

@media screen and (min-width: 253px) and (max-width: 767px) {
  .faq-content-row p {
    font-size: 11px;
  }
}

@media screen and (min-width: 768px) and (max-width: 1199px) {
  .faq-content-row img {
    width: 100%;
    height: 250px !important;
  }
}

@media screen and (min-width: 365px) and (max-width: 767px) {
  .faq-content-row img {
    height: 180px;
  }
}
