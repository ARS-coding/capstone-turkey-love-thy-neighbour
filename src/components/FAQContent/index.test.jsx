import React from "react";
import renderer from "react-test-renderer";
import FAQContent from ".";

describe("FAQContent Section", () =>
  it("renders Features section correctly", () => {
    const tree = renderer.create(<FAQContent />).toJSON();
    expect(tree).toMatchSnapshot();
  }));
