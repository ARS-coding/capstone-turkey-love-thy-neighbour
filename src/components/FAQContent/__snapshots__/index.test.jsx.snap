// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FAQContent Section renders Features section correctly 1`] = `
<div
  className="faq-content-container d-flex align-items-center container"
>
  <div
    className="faq-content-row m-0 row"
  >
    <div
      className="first-column d-flex flex-column justify-content-center align-items-start col-md-6 col-sm-12 col-12"
    >
      <h2>
        FAQs
      </h2>
      <h5>
        Are meetings safe?
      </h5>
      <p>
        For this important topic, we have collaborated with popular cafes and places all around of Istanbul so that our neighbours would be able to meet with each other without worries.
      </p>
      <h5>
        What is the point?
      </h5>
      <p>
        More than 264 million people suffer from depression worldwide and with quarantine, it was not getting any better. That’s why it’s so important for us to solve the problem of people feeling more and more isolated as time progresses. We care about you, and would like you to care about your neighbour so that, we could be a stronger community.
      </p>
      <h5>
        How does all of this work?
      </h5>
      <p>
        With the data that you provide in your profile, we are carefully filtering neighbours that you may want to meet and network with.
      </p>
    </div>
    <div
      className="my-md-auto d-flex d-sm-flex d-md-block justify-content-center justify-content-sm-center justify-content-md-center col-md-6 col-sm-12 col-12"
    >
      <img
        alt="img"
        className="img-fluid"
        src="UndrawQuestion.png"
      />
    </div>
  </div>
</div>
`;
